index.html

```diff
@@ -187,7 +187,7 @@ <h4 class="lets-connect-message">LET'S CONNECT</h4>
             <li class="listItem"><img src="images/Disabled.png" alt="Disabled Arrow">Skills <hr> </li>
         </ul>
     </section>
-    <!--Footer section-->
+    <!-----Footer section----->
```

Comment:
- [ ] Your website is online, well done :+1: . However, there are some issues you need to address. Your desktop version is working well online, you need to adjust your mobile version online. Please add some `margin`  at the left to all your single project card because by sticking to the Figma template we help you to have a good User Experience empathizing with users to ensure their journey through your website is both functional and pleasurable. :smile: 

### Your project
![Screenshot from 2022-07-11 14-46-51](https://user-images.githubusercontent.com/6615394/178294281-b145613c-b6f7-4eea-86c9-05331731487e.png)

### Figma template
![Screenshot from 2022-07-11 14-40-48](https://user-images.githubusercontent.com/6615394/178294285-e9061e01-c882-424e-ad95-cbc2dbb24b8b.png)

- [ ] I would like to recommend also working on the online mobile version, especially the **About myself** section behavior. Please make sure that all the elements are aligned as defined in the Figma template because it is very important to give your users a very good user experience and also stick to the Figma template as much as possible. :+1: 

### Your project
![Screenshot from 2022-07-11 15-06-09](https://user-images.githubusercontent.com/6615394/178296939-78d29505-2667-4ee1-8522-c523cf4d74ee.png)

### Figma template
![Screenshot from 2022-07-11 15-05-49](https://user-images.githubusercontent.com/6615394/178296951-cc6c8c96-e1aa-4eaa-a46e-c6e6158085f0.png)


- [ ] Please use adjust the `padding` for your `ul` and `li` for categories in the "Work section" Html, CSS, and Javascript list items because it is important to stick to the Figma template as much as possible. :+1: 

- [ ] Please use the same background color `white` for all your button when it is enabled :smile: 

### Your project
![Screenshot from 2022-07-11 15-13-53](https://user-images.githubusercontent.com/6615394/178298962-5cea7f95-e56d-4a85-9efe-fd965374134c.png)

### Figma template
![Screenshot from 2022-07-11 15-13-26](https://user-images.githubusercontent.com/6615394/178298968-b96f8ed1-1a6b-4bd5-bc2a-e1c818aa7d70.png)